### PR TITLE
Update About Dolphin Screen

### PR DESCRIPTION
**Pull request by RisingFog**
Based off mockup found here: https://code.google.com/p/dolphin-emu/issues/detail?id=7106#c4

image

### CODE REVIEW

**Comment by magumagu** [comment]
Using wxLIGHT_GREY is a bad idea.  Rule of thumb: if you aren't setting the background color explicitly, you shouldn't set the foreground color explicitly.  Maybe there's a "disabled" color or something like that you could use.

Similarly, a transparent logo against an unknown background color isn't a good idea.

The check for updates text should either be a link/button or not there at all.

**Comment by RisingFog** [comment]
@magumagu wxNullColour for the background perhaps?

**Comment by magumagu** [comment]
wxNullColour isn't a color.  The problem is that we can end up with results like this:
themefun

File: Source/Core/DolphinWX/AboutDolphin.cpp [hunk 1]
```diff
@@ -29,50 +29,53 @@ AboutDolphin::AboutDolphin(wxWindow *parent, wxWindowID id,
...
+	const wxString DolphinText = _("Dolphin Emulator");
+	const wxString RevisionText = scm_desc_str;
+	const wxString CopyrightText = _("� 2003-2015+ Dolphin Team. \"GameCube\" and \"Wii\" are trademarks of Nintendo. Dolphin is not affiliated with Nintendo in any way.");
+	const wxString Text = wxString::Format(_("Branch: %s\n"
+		"Revision: %s\n"
+		"Compiled: %s @ %s\n"
+		"\n"
+		"Dolphin is a free and open source GameCube/Wii emulator.\n"
+		"\n"
+		"This software is for educational purposes only and should not be\n"
```
Comment (JosJuice) [inline Source/Core/DolphinWX/AboutDolphin.cpp]: Why "for educational purposes only"? Isn't playing games you own a perfectly valid use for emulators?

Comment (lioncash) [inline Source/Core/DolphinWX/AboutDolphin.cpp]: Originally this used to be part of the old description as well, however it was removed for exactly that reason. Dunno why it's back again.

**Comment by BhaaLseN** [comment]
Hmm, any opinions about removing the rest of the old notice with the thanks and all that stuff?
Its not in the readme or elsewhere, and would be gone completely if we remove it from there.

**Comment by RisingFog** [comment]
@BhaaLseN might be a good idea to move it into the readme?

**Comment by neobrain** [comment]
@BhaaLseN just my 2c: I think those "thanks and all that stuff" is quite out-of-date in the sense that other people have put in equal amounts of work into the project since then, yet aren't mentioned. In other words: Why limit the list of mentioned people to the state of ~7 years ago?

That said, I don't think it really serves much of a point at all.

**Comment by neobrain** [comment]
Also, something to think about: This patch introduces inconsistent branding. Unless I'm mistaken, the project is being referred to as "Dolphin" throughout the whole GUI, while the about dialog uses the name "Dolphin Emulator" (despite actually calling it "Dolphin" in its title bar!)

**Comment by mrgreywater** [comment]
I also like the links in the [original mockup](https://i.imgur.com/QEwuzvw.png). They would allow instant access to the full license and website. Additionally the mockup uses different fonts and background colors, which visually improve the overall look of the window. The "Ok" button isn't really needed.

**Comment by neobrain** [comment]
@mrgreywater I'm personally fine with having a button, however I think this kind of dialog usually gets a "Close" button rather than an "Ok" button. At least that's what all KDE about dialogs use.

**Comment by RisingFog** [comment]
@neobrain it originally said just Dolphin, but MaJoR suggested having it as Dolphin Emulator instead.

**Comment by mrgreywater** [comment]
this mockup was heavily oriented at the firefox about page, which looks like this (gnome 3 and windows 8.1)[http://imgur.com/a/dt1Iw]. I'm not saying it has to be done this way, but they don't have any button, and i like their design.

**Comment by BhaaLseN** [comment]
Thought I'd bring it up as a point, since its apparent compared to the old about screen. Other than that, I'm pretty much +/-0 on it for the reasons mentioned.

**Comment by lioncash** [comment]
I agree with @mrgreywater. No buttons looks cleaner compared to a UI with buttons on it. If hyperlinked text is to be added (through later decision, or whatnot), wxWidgets provides a wxHyperlinkCtrl for this.

File: Source/Core/DolphinWX/AboutDolphin.cpp [hunk 2]
```diff
@@ -29,50 +29,52 @@ AboutDolphin::AboutDolphin(wxWindow *parent, wxWindowID id,
...
-				"you do not legally own."),
-		scm_desc_str, scm_branch_str, scm_rev_git_str, __DATE__, __TIME__);
+	const wxString DolphinText = _("Dolphin Emulator");
+	const wxString RevisionText = scm_desc_str;
+	const wxString CopyrightText = _("© 2003-2015+ Dolphin Team. \"GameCube\" and \"Wii\" are trademarks of Nintendo. Dolphin is not affiliated with Nintendo in any way.");
+	const wxString Text = wxString::Format(_("Branch: %s\n"
+		"Revision: %s\n"
+		"Compiled: %s @ %s\n"
+		"\n"
+		"Dolphin is a free and open source GameCube/Wii emulator.\n"
```
Comment (Buddybenj) [inline Source/Core/DolphinWX/AboutDolphin.cpp]: Generally, we put a space before and after the slash (i.e. GameCube / Wii). We do so on the readme and the GitHub repository description.

**Comment by Pringo** [comment]
Hey, interesting to see my mockup being used. I'd help out with this but I know nothing about wxWidgets.

Regardless I have some thoughts on the current window:

- There seems to be some noticeable aliasing in the "Dolphin" text (at least in Windows 8.1). Is it possible to fix this?
- Some extra padding around the logo and the text would be nice like it is in the mockup. Right now things look a bit cramped.
- Is it possible to do the lighter font for the Branch/Revision/Compiled sections (and maybe copyright too)? If not is there some other way to de-emphasize it? I find those lines make the window look a bit messy and they generally aren't all that important for the average user.
- It would be nice to add back at least some of the links from the mockup. Personally I'd like to see the check for updates link (https://dolphin-emu.org/download/) and the authors link (https://github.com/dolphin-emu/dolphin/graphs/contributors). I think it would be best to have a dev list what links are important here. Some of the ones I chose for the mockup were a bit arbitrary.
- The logo seems to have some aliasing around the edges too. Is this a scaling issue? Also I added a bit of shading to my version if you want to use that (I uploaded it here: https://i.imgur.com/HQ49dVK.png). I guess that's mostly a stylistic choice more than anything. I still have all my assets from the mockup if you want anything else.
- Finally I think a dev should sign off on the copyright notice/legal statement changes before this goes through.

**Comment by RisingFog** [comment]
@Pringo for the anti-aliasing in both the logo and the text, there's nothing that I can do about that as far as I know. 

For the branch/copyright font colours, it was initially light gray, but it causes issues with people who use anything other than default themes for the operating system.

As for the links, we already link to the GitHub and the main website in the Help tab. It seems unnecessary to also link it in the about page.

**Comment by mrgreywater** [comment]
@Pringo light gray is fine, as long as you force the background to white

File: Source/Core/DolphinWX/AboutDolphin.cpp [hunk 3]
```diff
@@ -29,51 +29,51 @@ AboutDolphin::AboutDolphin(wxWindow *parent, wxWindowID id,
...
-				"Thanks to hcs/destop for their GC ADPCM decoder.\n"
-				"\n"
-				"We are not affiliated with Nintendo in any way.\n"
-				"GameCube and Wii are trademarks of Nintendo.\n"
-				"The emulator should not be used to play games\n"
-				"you do not legally own."),
-		scm_desc_str, scm_branch_str, scm_rev_git_str, __DATE__, __TIME__);
+	const wxString DolphinText = _("Dolphin");
+	const wxString RevisionText = scm_desc_str;
+	const wxString CopyrightText = _("(c) 2003-2015+ Dolphin Team. \"GameCube\" and \"Wii\" are trademarks of Nintendo. Dolphin is not affiliated with Nintendo in any way.");
```
Comment (Buddybenj) [inline Source/Core/DolphinWX/AboutDolphin.cpp]: If we're not going to use the copyright symbol, I think we should use a capital C in parentheses.

Comment (lioncash) [inline Source/Core/DolphinWX/AboutDolphin.cpp]: Lower-case C is fine (and is the usual way it's done)

Comment (bb010g) [inline Source/Core/DolphinWX/AboutDolphin.cpp]: Why not the Unicode ©? http://docs.wxwidgets.org/trunk/overview_unicode.html

Comment (lioncash) [inline Source/Core/DolphinWX/AboutDolphin.cpp]: This was already implied in @Buddybenj's "If we're not going to use the copyright symbol"

**Comment by delroth** [comment]
If we can get it implemented, I really like @mrgreywater's version.

**Comment by RisingFog** [comment]
@delroth that was the original mockup that this about screen was based off of. We already have the dolphin website in the help tab, as well as the github. Didn't see a need to implement something that was already implemented. Perhaps it could be moved into there exclusively?

**Comment by delroth** [comment]
I think it looks better with the 3 links at the bottom personally, even if it's slightly redundant.

File: Source/Core/DolphinWX/AboutDolphin.cpp [hunk 4]
```diff
@@ -29,51 +29,51 @@ AboutDolphin::AboutDolphin(wxWindow *parent, wxWindowID id,
...
+	const wxString Text = wxString::Format(_("Branch: %s\n"
+		"Revision: %s\n"
+		"Compiled: %s @ %s\n"
+		"\n"
+		"Dolphin is a free and open source GameCube / Wii emulator.\n"
```
Comment (lioncash) [inline Source/Core/DolphinWX/AboutDolphin.cpp]: "Dolphin is a free and open source GameCube and Wii emulator.\n"

File: Source/Core/DolphinWX/AboutDolphin.cpp (hunk 3 above)
Comment (Buddybenj) [inline Source/Core/DolphinWX/AboutDolphin.cpp]: We should probably just say the first year of publication (2003). It's more common, and we wouldn't have to update it every year. Also, IMO spelling out "Copyright" looks prettier than "(c)".

File: Source/Core/DolphinWX/AboutDolphin.cpp [hunk 5]
```diff
@@ -29,51 +30,74 @@ AboutDolphin::AboutDolphin(wxWindow *parent, wxWindowID id,
...
-		scm_desc_str, scm_branch_str, scm_rev_git_str, __DATE__, __TIME__);
+	const wxString DolphinText = _("Dolphin");
+	const wxString RevisionText = scm_desc_str;
+	const wxString CopyrightText = _("(c) 2003-2015+ Dolphin Team. \"GameCube\" and \"Wii\" are trademarks of Nintendo. Dolphin is not affiliated with Nintendo in any way.");
+	const wxString BranchText = wxString::Format(_("Branch: %s\n"
+		"Revision: %s\n"
+		"Compiled: %s @ %s"), scm_branch_str, scm_rev_git_str, __DATE__, __TIME__);
+	const wxString CheckUpdateText = _("Check for updates: ");
+	const wxString Text = _("\n"
+		"Dolphin is a free and open source GameCube and Wii emulator.\n"
```
Comment (Buddybenj) [inline Source/Core/DolphinWX/AboutDolphin.cpp]: free and open-source

**Comment by Pringo** [comment]
Some thoughts on the latest iteration:

- There should be some padding on the right side of the window.
Dolphin About Screen Padding
The padding marked by the red columns here should be consistent. The first two are mostly fine but there isn't really any space between the text and the window border on the right. The "This software should not be..." sentence should be two lines.
- I think there should also be some extra vertical padding after the links and before the copyright notice as that part looks a bit cramped.
- As @bb010g mentioned it would look nicer to use a Unicode © in the copyright line rather than (c).
- Is it possible to do the two tone background like it is in the mockup? It really helps compartmentalize the window and makes it look less bland overall. @mrgreywater previously mentioned you can force the background colour. I'm not sure if this would cause any problems or not. This would also help with implementing the lighter text.
- Regardless if the previous suggestion is implemented I still think something should be done to clean up that Branch/Revision/Compiled section.
- I think a heavier font should be used for the "Dolphin" title text. I don't really like how I did it in the mockup either. Perhaps someone with typography experience could suggest something.
- This one isn't really important but is it possible to remove the underlines for the links and have them appear on mouseover? The Firefox about screen does this and I think it's a bit nicer looking.

**Comment by Sonicadvance1** [comment]
Forcing a background colour is pretty terrible. Especially with desktop environments that support more theming options than what Windows offers.
It's a bit jolting when in a dark theme, and Dolphin switches over to some other colour that was selected by the developers rather than the theme the user is running.

**Comment by neobrain** [comment]
I second @Sonicadvance1's opinion. Hardcoding background colors is a bad idea.

File: Source/Core/DolphinWX/AboutDolphin.cpp [hunk 6]
```diff
@@ -29,51 +30,73 @@ AboutDolphin::AboutDolphin(wxWindow *parent, wxWindowID id,
...
+	CopyrightFont.SetPointSize(7);
+	Copyright->SetFont(CopyrightFont);
+
+	wxBoxSizer* const sCheckUpdates = new wxBoxSizer(wxHORIZONTAL);
+	sCheckUpdates->Add(Update);
+	sCheckUpdates->Add(Download);
 
 	wxBoxSizer* const sInfo = new wxBoxSizer(wxVERTICAL);
-	sInfo->Add(Message, 1, wxEXPAND | wxALL, 5);
+	sInfo->Add(Dolphin, 0, wxEXPAND | wxALL, 0);
```
Comment (lioncash) [inline Source/Core/DolphinWX/AboutDolphin.cpp]: Zero can be omitted here.

File: Source/Core/DolphinWX/AboutDolphin.cpp [hunk 7]
```diff
@@ -29,51 +30,73 @@ AboutDolphin::AboutDolphin(wxWindow *parent, wxWindowID id,
...
+	sInfo->Add(Revision, 0, wxEXPAND | wxALL, 2);
+	sInfo->Add(Branch, 0, wxEXPAND | wxALL, 2);
+	sInfo->Add(sCheckUpdates, 0, wxEXPAND | wxALL, 2);
+	sInfo->Add(Message, 0, wxEXPAND | wxALL, 2);
 
 	wxBoxSizer* const sMainHor = new wxBoxSizer(wxHORIZONTAL);
-	sMainHor->Add(sbDolphinLogo, 0, wxEXPAND | wxALL, 5);
+	sMainHor->Add(sbDolphinLogo, 0, wxEXPAND | wxALL, 0);
```
Comment (lioncash) [inline Source/Core/DolphinWX/AboutDolphin.cpp]: Here as well

File: Source/Core/DolphinWX/AboutDolphin.cpp [hunk 8]
```diff
@@ -29,51 +30,73 @@ AboutDolphin::AboutDolphin(wxWindow *parent, wxWindowID id,
...
 	sMainHor->Add(sInfo);
 
+	wxGridSizer* const sLinks = new wxGridSizer(3, 1, 150);
+	sLinks->Add(License, 0, wxEXPAND | wxALL, 0);
+	sLinks->Add(Authors, 0, wxEXPAND | wxALL, 0);
+	sLinks->Add(Support, 0, wxEXPAND | wxALL, 0);
```
Comment (lioncash) [inline Source/Core/DolphinWX/AboutDolphin.cpp]: And for these three

File: Source/Core/DolphinWX/AboutDolphin.cpp [hunk 9]
```diff
@@ -29,51 +30,73 @@ AboutDolphin::AboutDolphin(wxWindow *parent, wxWindowID id,
...
+
+	wxBoxSizer* const sFooter = new wxBoxSizer(wxVERTICAL);
+	sFooter->Add(sLinks, 0, wxALIGN_CENTER_HORIZONTAL, 5);
+	sFooter->AddSpacer(10);
+	sFooter->Add(Copyright, 0, wxALIGN_BOTTOM | wxALIGN_CENTER, 0);
```
Comment (lioncash) [inline Source/Core/DolphinWX/AboutDolphin.cpp]: and this one.

**Comment by Pringo** [comment]
Given the stated preference of not hardcoding any colours I tried to modify the existing design to accommodate that constraint while still looking decent. Here's my updated mockup:

Updated Dolphin About Screen

Changes:

- Made non-link text all one colour.
- Made background all one colour.
- De-emphasized Branch / Revision / Compiled section by reducing font size (comparable to the copyright line).
- Moved the Authors / License / Support links to be aligned with the rest of the text due to the removal of the two tone background. Also added dividers between them.
- Reduced window height as a response to the change in design.

Perhaps this might be useful for the final design.

**Comment by comex** [comment]
Please remove the claim that the software is for educational purposes only.  (As I said when I got rid of it a while back, there is no legal value in making patently false claims, so it is just embarrassing.)  The second part of the sentence is fine though.

**Comment by Pringo** [comment]
@comex Fixed.

**Comment by BhaaLseN** [comment]
If its easily possible, can you make the Branch/Revision/Compiled bit (and possibly the version before it) selectable, so the user can just copy/paste it out from there if he needs to (ex. when creating an issue report or smth like that)?

**Comment by Pringo** [comment]
@BhaaLseN I think it's more important to have the version number selectable rather than the Branch/Revision/Compiled information. The version number is used frequently for things like issue reports whereas the Branch/Revision/Compiled information isn't.

**Comment by BhaaLseN** [comment]
Agreed, but typing "4.0.2" by hand is easier than "2879cbd2b5648bf896ed521628284bd4d96c3d4f". Given the option, I'd want it all if possible :)
The latter is ofc more useful for those who use development builds.

As alternative, put a button/link somewhere that copies the interresting (from a developer perspective) values directly to the clipboard; but I'd imagine that a bit harder than just putting a R/O textbox there instead of a label.

**Comment by kkkllleee** [comment]
Are there any other graphical interfaces in need of updating?

**Comment by RisingFog** [comment]
The revision number as well as the branch/hash/compiled date are now copy/pasteable

**Comment by Pringo** [comment]
@RisingFog I just tried out the latest version and it looks a bit better but I still think there are some issues to address. The URL underlining is too bad but I don't think it's really all that big of a problem.

Outstanding Issues:

- Is the Unicode © possible for the copyright line or not? It would look better than the current solution.
- Is the smaller font size for the Branch/Revision/Compiled section from my latest mockup doable?
- Can you change the Authors / License / Support links to be like the latest mockup. As the developers stated that they don't want the two colour background design I think these links really need to be changed as the current look doesn't make much sense without it.
- The vertical spacing between the different text boxes is inconsistent. See:
About Screen Text Vertical Spacing
These should be uniform.
- You switched back from the shadowed logo to the previous one with no shadow and noticeable aliasing. Was this intentional or a mistake?
- This is less important but I still think the title font / size could be improved.

New Issues:

- The version number is no longer aligned with the rest of the text. Can you fix this? See:
Text Alignment
The title should also be shifted a couple pixels to the left.
- The selectable text is nice but now there's a blinking text cursor before the version number by default. Can this be removed?

**Comment by BhaaLseN** [comment]
Ah, I noticed the aliasing but didn't really follow up on that...so it wasn't just my eyes :S

**Comment by RisingFog** [comment]
@Pringo 
- Unicode © breaks when placed in GitHub and then pulled/compiled (at least for me)
- Smaller branch/revision stuff is definitely do-able
- I'll look into making the URLs match the current mockup
- Spacing has been a bit of an issue, I'll look into it as well
- MaJoR requested the logo to not have the dropdown. I still have no resolution for the aliasing
- The font could potentially change, but having it bold at the current size makes it too thick
- I'll look into the alignment issue
- There is a native way to hide it in Windows, but it will not compile in any other OS. Perhaps I could make it a different colour that blends in?

**Comment by magumagu** [comment]
You should be able to write the copyright sign as `L"\u00A9"`.  It needs to be a wide string so the compiler encodes it correctly on Windows. Escaping it should avoid any source code encoding issues.

**Comment by Buddybenj** [comment]
@RisingFog I don't know if you saw my comment above, but do you mind changing "free and open source" to "free and open-source" (hyphen)?

**Comment by RisingFog** [comment]
image

What I've been able to work with

**Comment by kkkllleee** [comment]
Wouldn't it be nice to include links to here and the forum? You do have a lot of space there.

**Comment by BhaaLseN** [comment]
I can live with that, and personally I don't mind the blinking cursor (its selectable after all, so there's a cursor, why not) - only thing that surprises me is that the box has focus by default, which it probably shouldn't. But we had this issue of the text being selected by default, and I don't know if we could do anything against that.

**Comment by Sonicadvance1** [comment]
God damn Windows, do some subpixel blending on that D.
LGTM

**Comment by lioncash** [comment]
I can just make a follow-up commit that fixes the minor style-related corner-cases. So if it's something minor don't worry about it.

**Comment by RisingFog** [comment]
@BhaaLseN  I had it focus on the version number in an attempt to hide the caret with what turned out to be a Windows only feature. I've since made it focus on a non-selectable form object so the blinking cursor does not appear until clicked on the text control.

@Sonicadvance1 I have to actually push the new changes I made yesterday to it, forgot to do that before I left for work. Should be pushed in 3-4 hours. Once that's done it should be good to go.

**Comment by Pringo** [comment]
@RisingFog Looks better. It looks to me like there's still some issues with the text alignment though:

Text Alignment

These text boxes should all be left aligned together. The Dolphin title, version number, and Branch/Revision/Compiled look they they need to be shifted to the left by varying amounts. Finally I think there could also be a bit more vertical space added between the Authors / License / Support links and the copyright line (at least as much as the spacing between the other sections).

Otherwise LGTM.

**Comment by RisingFog** [comment]
@Pringo For some reason, as soon as I apply a styling (bold, font size changes), it knocks it out of whack. I don't really have a solution for it at the moment.

Anyways, here's the final look:
image

**Comment by MaJoRoesch** [comment]
My only problem is that the name of the emulator is "Dolphin Emulator", since Dolphin is so generic.

That and, like the firefox about that this is based on, I'd like the Dolphin font to be unique  (I usually use "Yanone Kaffeesatz"). But that would be yet another image to include... If you are open to it I can provide it though.

And the aliasing still bugs me. :( But I guess that can't be helped, stupid wx. Add it to the list of things Qt will fix?

Other than those things, I like it. It is MUCH better organized than the old about page, and doesn't have that silly cartoonish logo! It displays useful information about Dolphin cleanly and simply. That's what it's supposed to be!

**Comment by MaJoRoesch** [comment]
Oh shoot, I almost forgot! @RisingFog did you set up a @2x version of the logo and has anyone tried this on hidpi capable systems?

File: Source/Core/DolphinWX/AboutDolphin.cpp [hunk 10]
```diff
@@ -29,51 +31,94 @@ AboutDolphin::AboutDolphin(wxWindow *parent, wxWindowID id,
 	wxStaticBitmap* const sbDolphinLogo = new wxStaticBitmap(this, wxID_ANY,
```
Comment (lioncash) [inline Source/Core/DolphinWX/AboutDolphin.cpp]: This should actually be a wxGenericStaticBitmap, and is recommended in wx documentation for large images. You can get access to that by changing `#include <wx/statbmp.h> to `#include <wx/generic/statbmpg.h>` (don't forget to move it after the windowid.h include so you don't trip the lint check).

**Comment by Pringo** [comment]
@MaJoRoesch The Dolphin naming was actually addressed earlier in this PR by @neobrain. The rest of the program GUI uses the name "Dolphin" rather than "Dolphin Emulator" so changing it in the About screen would be inconsistent. My understanding is that "Dolphin" is in fact the complete program name and the term emulator is brought in when it is more necessary to differentiate between the various other Dolphin programs out there (browser, etc). To be honest this hasn't been handled all that well historically as the two terms are used interchangeably in a number of places (although dealing with that is well beyond the scope of this PR).

Also I do like the look of that Yanone Kaffeesatz typeface compared to the current one. I would support switching to it.

**Comment by neobrain** [comment]
> My only problem is that the name of the emulator is "Dolphin Emulator", since Dolphin is so generic.

Just that it isn't. "Dolphin Emulator" is a name which is used on the main website (not on the wiki, or the forums, or the program itself, or (most if not all) other places). So it's really the website using the wrong name, if anything, and it's doing so mostly for the purpose of SEO afaik.

Just my 2c: "Dolphin Emulator" is a terrible name, really. I can't really express in words why that is (part of it is that I'd prefer people saying "I'm using the Dolphin E/emulator" instead of "I'm using the emulator Dolphin"). Either way, if you'd like to put a name change up to discussion I could elaborate, but it's kind of besides the point here, since for this particular PR it suffices to say that "Dolphin" is the most commonly used brand name we have.

**Comment by RisingFog** [comment]
Slightly going off topic, the most recent commit fixes the antialiasing on
the logo, so that should no longer be a problem.

**Comment by Buddybenj** [comment]
LGTM

**Comment by RisingFog** [comment]
bump?

**Comment by MaJoRoesch** [comment]
@RisingFog 2x version of the logo?

**Comment by degasus** [comment]
This PR is a bit broken on linux: First there is a "iCCP: known incorrect sRGB profile" error message, so the PNG is likely broken.
On the second hand, there is no close button at all and both the version number and the revision details have a wrong background color.

**Comment by endrift** [comment]
I tried to fix some of that in #2324 on OS X--it doesn't fix the background color issue (which is there because the change is using a text *field* for a text *label*), but it fixes the PNG, crops it so there isn't padding *in the actual image data* and adds a 2x version. I don't know how much that affects Linux, though; I don't have a 2x display on Linux and I don't know if this affects computers with 1x displays.

**Comment by Icekhaos** [comment]
PR 2016: A Cautionary Tale of WX

Besides that, let's hope Qt gets ready soonish.

**Pull request merged**